proto/interchainqueries/params.proto

```diff
@@ -18,4 +18,8 @@ message Params {
     (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins",
     (gogoproto.nullable) = false
   ];
+
+  // Amount of tx hashes to be removed during a single EndBlock. Can vary to balance
+  // between network cleaning speed and EndBlock duration. A zero value means no limit.
+  uint64 tx_query_removal_limit = 3;
```

Comment:
Why is it 3 by default?
You said 5k is good enough?

Reply:
3 is the number of the field in the proto msg

Reply:
Oh lol, i need to get some sleep